docs/pluggable_streams.rst

```diff
@@ -144,3 +144,51 @@ The properties for the thresholds are as follows:
 
 
 An example of this implementation can be found in the `KafkaConsumerFactory <https://github.com/apache/incubator-pinot/blob/master/pinot-core/src/main/java/org/apache/pinot/core/realtime/impl/kafka/KafkaConsumerFactory.java>`_, which is an implementation for the kafka stream.
+
+
+Kafka 2.x Plugin
+^^^^^^^^^^^^^^^^
+
+Pinot provides stream plugin support for Kafka 2.x version.
+Although the version used in this implementation is kafka 2.0.0, it's possible to compile it with higher kafka lib version, e.g. 2.1.1.
+
+How to build and release Pinot package with Kafka 2.x connector
+---------------------------------------------------------------
+
+.. code-block:: none
+
+  mvn clean package -DskipTests -Pbin-dist -Dkafka.version=2.0
+
+How to use Kafka 2.x connector
```

Comment:
Does kafka-2.x not support LLC? Or, did we skip the documentation?

Reply:
Both hlc and llc are supported. I will add document soon.